processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -59,7 +59,7 @@
   public static final boolean DEFAULT_USE_CACHE = true;
   public static final boolean DEFAULT_POPULATE_RESULTLEVEL_CACHE = true;
   public static final boolean DEFAULT_USE_RESULTLEVEL_CACHE = true;
-  public static final Vectorize DEFAULT_VECTORIZE = Vectorize.FALSE;
+  public static final Vectorize DEFAULT_VECTORIZE = Vectorize.TRUE;
```

Comment:
Just to see if anything else fail. Will make this change in a separate PR with relevant doc change